.stickler.yml

```diff
@@ -0,0 +1,16 @@
+linters:
+  rubocop:
+    display_cop_names: true
+    fixer: true
```

Comment:
Please don't do this. Nothing wrong about it, but for learning purposes you are not supposed to do it. You are supposed to fix the issues by your selves.